CHANGELOG.md

```diff
@@ -5,11 +5,10 @@ All notable changes to PowerModelsMCDC.jl will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-
-### Fixed
+## prep. [0.2.0] - 2024-08-07
```

Comment:
As this is a Markdown file, the text in square brackets (like `[0.1.1]`) constitutes at the same time the *text* and the *label* of a [reference-style link](https://daringfireball.net/projects/markdown/syntax#link). For such a link to work, the same *label* and the *linked address* must be specified somewhere else in the same file. You can find them at the end of the file for both the previous releases and the unreleased changes. When registering a new release, the maintainer must take care of updating the links as well. The reason for using reference-style links instead of inline links is that we don't want the linked address to end up in section titles.